components/test/src/main/java/org/trellisldp/test/LdpBasicContainerTests.java

```diff
@@ -283,17 +272,18 @@ default void testCreateContainerWithSlug() {
     @DisplayName("Test deleting a basic container")
     default void testDeleteContainer() {
         final RDF rdf = getInstance();
-        final EntityTag etag;
-
         final String childResource;
 
+        final EntityTag initialETag = getETag(getContainerLocation());
+
         try (final Response res = target(getContainerLocation()).request().post(entity("", TEXT_TURTLE))) {
             assertAll("Check for an LDP-RS", checkRdfResponse(res, LDP.RDFSource, null));
             childResource = res.getLocation().toString();
         }
 
-        meanwhile();
+        await().until(() -> !initialETag.equals(getETag(getContainerLocation())));
```

Comment:
Is there a timeout on this? I worry a bit about tests that hang indefinitely... but maybe it's not an issue?

Reply:
The default is 10 seconds. After that, the test will fail.